src/desktop/components/main_layout/tti-init.js

```diff
@@ -0,0 +1,11 @@
+function ttiInit() {
```

Comment:
You should probably could just unwrap this. The function wrapper seems a little unnecessary.